### PR TITLE
Add certificate documentation

### PR DESCRIPTION
**Pull request by sharmilarama**
# Description

This PR adds documentation for TLS certificate requirements.

# Issues

List the issues impacted by this PR:

| Issue ID |
| -------- |
| #124 |

# Checklist:

- [x] I have performed a self-review of my own changes.

### CODE REVIEW

**Comment by ioeicicdsvc** [comment]
@sharmilarama: Thank you for submitting your pull request!

Please make sure you have followed our contributing guidelines. We will review it as soon as possible.

------------------------------------------------------------------------------------------------------------------------
We want your feedback! If you have any questions or suggestions regarding our contributing process/workflow, please reach out to us at karavi@dell.com.

**Review by alikdell — DISMISSED** [review]

File: docs/GETTING_STARTED_GUIDE.md [hunk 1]
```diff
@@ -85,7 +85,8 @@ In the above template, `DNS_host_name` refers to the host name of the system in
 nslookup <IP_address>
 ```
 
-3. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS_host_name` is also required. All traffic from `grpc.DNS_host_name` needs to be routed to `DNS_host_name` address, this can be configured by adding a new DNS entry for `grpc.DNS_host_name` or providing a temporary path in the `/etc/hosts` file.
+3. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS_host_name` is also required. All traffic from `grpc.DNS_host_name` needs to be routed to `DNS_host_name` address, this can be configured by adding a new DNS entry for `grpc.DNS_host_name` or providing a temporary path in the `/etc/hosts` file.  
+NOTE: The certificate/key pair provided in `crtFile` and `keyFile` should be valid for both the `DNS_host_name` and the `grpc.DNS_host_name` address.
```
Comment (hoppea2) [inline docs/GETTING_STARTED_GUIDE.md:89]: Would it make sense to provide an example?

Comment (sharmilarama) [reply docs/GETTING_STARTED_GUIDE.md:89]: The cert creation can be done in different ways, not sure if we want to specify one command as an example.

**Review by LJ-Software — DISMISSED** [review]

File: docs/GETTING_STARTED_GUIDE.md (hunk 1 above)
Comment (hoppea2) [reply docs/GETTING_STARTED_GUIDE.md:89]: Let's provide one example

Comment (sharmilarama) [reply docs/GETTING_STARTED_GUIDE.md]: Fixed.

**Review by alikdell — APPROVED** [review]

**Review by hoppea2 — APPROVED** [review]

**Pull request merged**